src/store/users/api.ts

```diff
@@ -0,0 +1,7 @@
+import { get } from '../../lib/api/rest';
+
+export async function searchMyNetworks(channelId: string): Promise<any> {
```

Comment:
this needs to be named based on what it's actually doing. let's also pull it into the channels api, as we're fetching user for a channel, not users for a network.